### PR TITLE
Added docker images document

### PR DESCRIPTION
**Pull request by bhosale**
Added a document for creating image, building the image and pushing it to docker hub.

### CODE REVIEW

**Comment by kumaran-is** [comment]
@bhosale: The root  folder   readme.md  is empty. Can we have prescription link to this docker image docs

File: docs/docker/docker-images.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# Docker Images
```
Comment (kumaran-is) [inline docs/docker/docker-images.md:1]: Can we have table of contents . refer prescription of https://github.com/introduction-angular2/ng2-book-finder

File: docs/docker/docker-images.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
```
Comment (kumaran-is) [inline docs/docker/docker-images.md]: Can we have introduction to highlight/ over view  about this prescription 3 or 4 lines.  Now i have to go through whole prescription to understand what is this all about

File: docs/docker/docker-images.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+The below steps only provide instructions for creating and building docker images.
+
+### Build image using docker file
+The `docker build -t ng2-book-finder .` command takes the `Dockerfile` in the current directory, and builds an image called `ng2-book-finder` on your local machine.
+
+First Docker checks to make sure it has everything it needs to build.
+
+```
+ng2-book-finder (feature/docker-image) $ docker build -t ng2-book-finder .
```
Comment (kumaran-is) [inline docs/docker/docker-images.md]: There is no more feature branch.. refer to master branch -  Check  https://github.com/introduction-angular2/ng2-book-finder

File: docs/docker/docker-images.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+Status: Downloaded newer image for node:4.2-slim
+ ---> 264411d7f3f9
+```
+
+The below steps are performed for building the image.
+ - Expose the port
+ - set the required working directory
+ - copy required files for the image.
+ - Do `npm install`.
+ - And then compile the typescript to javascript.
```
Comment (kumaran-is) [inline docs/docker/docker-images.md:47]: Next time ,  just take the screen shot of this log  as image and show it ..  that would differentiate log statement/ console statement from doc

File: docs/docker/docker-images.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
+# Docker Images
+
+## Building your own image
+Please refer [Build your own image](https://docs.docker.com/mac/step_four/) for more reference.
+
+The below steps only provide instructions for creating and building docker images.
+
+### Build image using docker file
+The `docker build -t ng2-book-finder .` command takes the `Dockerfile` in the current directory, and builds an image called `ng2-book-finder` on your local machine.
```
Comment (kumaran-is) [inline docs/docker/docker-images.md:24]: add a link to ng2-book-finder  repo  https://github.com/introduction-angular2/ng2-book-finder

File: docs/docker/docker-images.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+```
+ng2-book-finder (feature/docker-image) $ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+ng2-book-finder     latest              3e957e2c6985        30 seconds ago      356.1 MB
+node                4.2-slim            264411d7f3f9        5 weeks ago         205.4 MB
+```
+
+## Managing the image in Docker Registry
+
+In `docker-hub` we have created an organization called as `introductiontodocker` and within that organization we are having a repository `ng2-book-finder`.
```
Comment (kumaran-is) [inline docs/docker/docker-images.md:107]: Can we have link to docker-hub  organization repository, so it will be easy to navigate and verify

**Comment by bhosale** [comment]
@kumaran-is : Updated as per the review comments

File: docs/docker/docker-images.md (hunk 6 above)
Comment (bhosale) [inline docs/docker/docker-images.md:107]: Added docker hub note.

File: docs/docker/docker-images.md (hunk 5 above)
Comment (bhosale) [inline docs/docker/docker-images.md:24]: added link to ng2-book-finder

File: docs/docker/docker-images.md (hunk 4 above)
Comment (bhosale) [inline docs/docker/docker-images.md:47]: OK. Next time.

File: docs/docker/docker-images.md (hunk 1 above)
Comment (bhosale) [inline docs/docker/docker-images.md:1]: Added TOC

**Comment by kumaran-is** [comment]
Approved

**Pull request merged**